mewarpx/mewarpx/emission.py

```diff
@@ -218,9 +210,9 @@ def __init__(self, emitter, species, npart,
             self.name = "fixed_injector_" + self.species.name
         self.unique_particles = unique_particles
 
-        print(("Fixed injection of {} particles, weight {}, every "
-            "{} timesteps.").format(
-                self.npart_total, self.weight, self.injectfreq))
+        print(f"Fixed injection of {self.npart_total} particles, " \
+            f"weight {self.weight}, every {self.injectfreq}" \
+            f"timesteps.")
```

Comment:
'\' is not necessary for strings inside of parentheses.
```suggestion
        print(f"Fixed injection of {self.npart_total} particles, "
              f"weight {self.weight}, every {self.injectfreq}"
              f"timesteps.")
```